docs/source/usage.rst

```diff
@@ -234,6 +244,17 @@ Disables logging during raw save. (I.e. for instance using loaddata)
 
 .. versionadded:: 2.2.0
 
+**AUDITLOG_CID_HEADER**
+
+The request header containing the Correlation ID value to use in all log entries created as a result of the request.
+
+.. versionadded:: 2.3.0
```

Comment:
The next release would be a major one. we have some pending PR which are breaking changes and I want to include them in next release
```suggestion
.. versionadded:: 3.0.0
```